### PR TITLE
VCP-17431 - Adding missing notifications back to server

### PR DESCRIPTION
**Pull request by rgvsilva**

### CODE REVIEW

File: deployment/updates/scripts/2022_01_05_admin_tags_equals_boolean_notification.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @package deployment
+ */
+require_once (__DIR__ . '/../../bootstrap.php');
+$script = realpath(dirname(__FILE__) . "/../../../tests/standAloneClient/exec.php");
+$newTemplateUpdate = realpath(dirname(__FILE__) . "/../../updates/scripts/xml/notifications/2022_01_05_admin_tags_equals_boolean_notification.template.xml");
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/2022_01_05_admin_tags_equals_boolean_notification.php:7]: need to remove .template here like in the rest pf the deployment scripts

File: deployment/updates/scripts/xml/2019_04_28_tagEqualsBooleanNotification.template.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (inbal-ben-david) [inline deployment/updates/scripts/xml/2019_04_28_tagEqualsBooleanNotification.template.xml:1]: i dont see deployment script for this template

Comment (rgvsilva) [reply deployment/updates/scripts/xml/2019_04_28_tagEqualsBooleanNotification.template.xml:1]: The deployment already existed. Just the XML didn't... Someone deleted the XML file between Quasar-17.2.0 and Quasar-17.3.0. I am just bringing it back.

File: deployment/updates/scripts/2022_01_05_admin_tags_equals_boolean_notification.php (hunk 1 above)
Comment (rgvsilva) [reply deployment/updates/scripts/2022_01_05_admin_tags_equals_boolean_notification.php:7]: Good catch. Fixing.

**Review by inbal-ben-david — APPROVED** [review]

**Pull request merged**